ingester/ingester.go

```diff
@@ -400,18 +275,30 @@ func (fi *FirehoseIngester) handleRecordCreate(
 		return nil
 	}
 
-	switch data := record.(type) {
-	case *bsky.FeedPost:
+	switch recordCollection {
+	case "app.bsky.feed.post":
+		data := &bsky.FeedPost{}
+		if err := json.Unmarshal(record, data); err != nil {
+			return fmt.Errorf("unmarshalling record: %w", err)
+		}
 		err := fi.handleFeedPostCreate(ctx, repoDID, recordUri, data)
 		if err != nil {
 			return fmt.Errorf("handling app.bsky.feed.post create: %w", err)
 		}
-	case *bsky.FeedLike:
+	case "app.bsky.feed.like":
+		data := &bsky.FeedLike{}
+		if err := json.Unmarshal(record, data); err != nil {
+			return fmt.Errorf("unmarshalling record: %w", err)
```

Comment:
**suggestion:** we could include the `recordCollection` type here, so something like

```suggestion
			return fmt.Errorf("unmarshalling app.bsky.feed.like: %w", err)
```